<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmailGui</class>
 <widget class="QDialog" name="EmailGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../Projects/Blink/icons/resources.qrc">
    <normaloff>:/png/marca-m.png</normaloff>:/png/marca-m.png</iconset>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>471</width>
     <height>258</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>451</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string> Mensagem</string>
    </property>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>431</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="LblFile">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>236</y>
      <width>440</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextBrowserInteraction</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>10</y>
      <width>431</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Para</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPara"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>50</y>
      <width>431</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Assunto</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditAssunto"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonEnviar">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Enviar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Cancel</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>textEdit</tabstop>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>pushButtonEnviar</tabstop>
  <tabstop>lineEditPara</tabstop>
  <tabstop>lineEditAssunto</tabstop>
 </tabstops>
 <resources>
  <include location="../../Projects/Blink/icons/resources.qrc"/>
  <include location="../../Projects/Blink/icons/resources.qrc"/>
 </resources>
 <connections/>
</ui>
